### ISSUE TITLE
PiT and PVT distillation issues

### ISSUE THREAD

**Issue by Sibgat-Ul** [sunshangquan/logit-standardardization-KD#26]
Hello, i was trying to train pit and pvt models, but got an error about the logits being none tho it works perfectly for DeiT model. While debugging i found that pvt and pit models does not has ```distill_logits```, only DeiT has that attribute (even on the original work). So how do i train them?
deit file (works fine, has distill_logits property for guidance loss): 
- ```https://github.com/lkhl/tiny-transformers/blob/main/pycls/models/transformers/deit.py```
- ```https://github.com/sunshangquan/logit-standardization-KD/blob/master/tiny-transformers/pycls/models/transformers/deit.py```

pvt file:
-  ```https://github.com/sunshangquan/logit-standardization-KD/blob/master/tiny-transformers/pycls/models/transformers/pvt.py```

guidance loss file:
- ```https://github.com/sunshangquan/logit-standardization-KD/blob/master/tiny-transformers/pycls/models/distill.py```

**Comment by sunshangquan** [comment]
Hi @Sibgat-Ul , the logits are the features after ```head```. 

For DeiT, it would be https://github.com/sunshangquan/logit-standardization-KD/blob/eb569ac3783f2eed16fbc2b021101175af4fb2f2/tiny-transformers/pycls/models/transformers/deit.py#L96

For PvT, it would be https://github.com/sunshangquan/logit-standardization-KD/blob/eb569ac3783f2eed16fbc2b021101175af4fb2f2/tiny-transformers/pycls/models/transformers/pvt.py#L236

For PiT, it would be https://github.com/sunshangquan/logit-standardization-KD/blob/eb569ac3783f2eed16fbc2b021101175af4fb2f2/tiny-transformers/pycls/models/transformers/pit.py#L171

It is my fault that I forgot to add the following two lines after the two lines of PvT and PiT respectively:

```self. distill_logits = x``` after L236 for PvT, and ```self. distill_logits = cls_token``` after L171 for PiT.

You could add them by yourself.

**Comment by Sibgat-Ul** [comment]
Thanks alot i will do that.
